src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -260,6 +261,10 @@ object NonNullableSetter {
   ): Encoder[NonNullableSetter[T]] = new Encoder[NonNullableSetter[T]] {
     final def apply(a: NonNullableSetter[T]): Json = a match {
       case SetValue(value) => Json.obj(("set", encodeT.apply(value)))
+      case AddRemove(None, None) =>
+        Json.obj(("add", Json.arr()), ("remove", Json.arr()))
+      case AddRemove(add, remove) =>
+        Json.obj(("add", encodeT.apply(add)), ("remove", encodeT.apply(remove)))
```

Comment:
I'll test how this part works in the API. Also, I can add a test on label updates.